task-list-rails/app/views/tasks/show.html.erb

```diff
@@ -2,18 +2,68 @@
   <table>
     <thead>
         <tr>
-          <th colspan="2" id="title">Task details</th>
+          <td colspan="2" id="title">Task details</td>
+        </tr>
+        <tr>
+          <td colspan="2" id=edit_link><%= link_to "edit?", "/tasks/#{@task.id}/edit" %></td>
         </tr>
       </thead>
       <tbody>
-      <% @task_info_hash.each do |key, value| %>
-        <% if !value.nil? && value != "" %>
+        <tr>
+          <td class="attribute">
+            <strong>name:</strong>
+          </td>
+          <td>
+            <%= @task.name %>
+          </td>
+        </tr>
+        <tr>
+          <td class="attribute">
+            <strong>description:</strong>
+          </td>
+          <% if @task.description == "" || @task.description.nil? %>
+            <td id="empty_description">
+              <em>empty</em>
+            </td>
+          <% else %>
+            <td>
+              <%= @task.description %>
+            </td>
+          <% end %>
+        </tr>
+        <tr>
+          <td class="attribute">
+            <strong>date created:</strong>
+          </td>
+          <td>
+            <%= @task.created_at.to_date %>
+          </td>
+        </tr>
+        <tr>
+          <td class="attribute">
+            <strong>completed?</strong>
+          </td>
+          <% if @task.completed %>
+            <td>yes</td>
+          <% else %>
+            <td>no</td>
+          <% end %>
+        </tr>
+        <% if @task.completed == true && !@task.completed_at.nil? %>
           <tr>
-            <td id="attribute"><strong><%= key %>:</strong></td>
-            <td><%= value %></td>
+            <td class="attribute">
+              <strong>date completed:</strong>
+            </td>
+            <td>
+              <%= "#{@task.completed_at.to_date}" %>
```

Comment:
This probably does not need to be using string interpolation, as the effect of `<%= %>` is to convert whatever value is inside the tag into a string anyways.